src/App.tsx

```diff
@@ -1,167 +1,21 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import Layout from './components/Layout';
```

Comment:
Ensure that the `Layout` component is correctly named and imported. Previously, there was a typo (`Laout`), so verify that this has been corrected in both the import statement and the component usage.